types/cvss/tsconfig.json

```
@@ -0,0 +1,24 @@
+{
+  "compilerOptions": {
+      "module": "commonjs",
+      "lib": [
+          "es5"
```

```suggestion
          "es6"
```
it's internal across DT packages, not really used when consuming the package. Do you have specific requirement here?

---

nah, again just copy-pasting from a previous contribution.